Presentation.Web/app/services/gridStateService.ts

```diff
@@ -405,7 +405,7 @@
                     columns.push({ persistId: column.persistId, index: i });
                 }
 
-                KendoFilterService.postConfigurationFromOrg(user.currentOrganizationId, overviewType, columns)
+                return KendoFilterService.postConfigurationFromOrg(user.currentOrganizationId, overviewType, columns)
```

Comment:
yeah the issue here is that it should actually tha t line 413 should be `return getGridOrganizationalConfiguration();`.
Otherwise the state will be not be set before actually reloading the grid. One could also argue that the local state should just be recomputed during initialization

Reply:
@a-naskret I committed at fix for this